krummstab/schemas/config-schema.json

```diff
@@ -138,6 +138,9 @@
           "pattern": "^\\.",
           "minLength": 2
         }
+      },
+      "command": {
+        "type": "string"
```

Comment:
- Make this a list of strings, starting with program command, with the following elements being arguments.
- List magic keywords as `{keyword}` so that we can use formatted strings directly to make the replacement later.